src/components/BookList.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import BookItem from './BookItem';
+
+function BookList() {
+  return (
+    <div className="book-list">
+      <BookItem />
+      <BookItem />
+      <BookItem />
```

Comment:
- [OPTIONAL]
   -  You only need to map the props to one BookItem component

Reply:
@AshakaE yes you're right.. I'll delete the extra components and use props by the next review. Thank you